clang/docs/ReleaseNotes.rst

```diff
@@ -159,6 +159,20 @@ AMDGPU Support
 X86 Support
 ^^^^^^^^^^^
 
+- The MMX vector intrinsic functions from ``*mmintrin.h`` which
+  operate on `__m64` vectors, such as ``_mm_add_pi8``, have been
+  reimplemented to use the SSE2 instruction-set and XMM registers
+  unconditionally. These intrinsics are therefore *no longer
+  supported* if MMX is enabled without SSE2 -- either from targeting
+  CPUs from the Pentium-MMX through the Pentium 3, or explicitly via
+  passing arguments such as ``-mmmx -mno-sse2``.
```

Comment:
@jyknight Please can you add a note that assembly / inline-assembly MMX code is still supported? Similar to what we had for the 3DNow! note in 19.x

Reply:
Done in b79568654e38a14ef921af932ed96abd8961b1ed.